aiida/cmdline/commands/cmd_group.py

```diff
@@ -353,3 +353,22 @@ def group_copy(source_group, destination_group):
     # Copy nodes
     dest_group.add_nodes(list(source_group.nodes))
     echo.echo_success('Nodes copied from group<{}> to group<{}>'.format(source_group.label, dest_group.label))
+
+
+@verdi_group.command('path')
+@click.option('-l', 'show_all', is_flag=True, default=False, help='Show all group paths')
```

Comment:
I think:
- by default just show the Path, with bold for non-virtual, without any table (like `ls` without options)
- `-l` should be `--long` like in `ls` and show the table. Also here, use bold for non-virtual, but ok to keep the column (but maybe move the `Virtual` column as the third one). I would also rename `Children` (that means `len(self.children)` -> `Groups contained`
- what now it's `-ls/show_all` should become `-R/--recursive` like `ls`

Reply:
Implemented these options, and some additional ones (see tests). Also, instead of `Groups contained` I have added `Sub-Groups`, which is a recursive count of all groups under that path. I think that will be more helpful.